config/theme.go

```diff
@@ -108,6 +108,161 @@ func NewBaseTheme(preset Preset) *Theme {
 			},
 		}
 
+	// Special Preset for Protanopias
+	case PresetProtDark:
+		return &Theme{
+			Default: MustParseColor("white"),
+			Base: ThemeBase{
+				Key:       MustParseColorSlice("#feb927 / white"),
```

Comment:
I think it's a bad idea to use white coloring in the keys. It makes it confused with the text color of the values in output like describe.

What if you had a weaker version of `#feb927` as the second key color?

Reply:
OK, changed
<img width="965" alt="image" src="https://github.com/kubecolor/kubecolor/assets/1110398/39b09ab9-5ed8-4327-be83-3a292abfbe38">

Reply:
That looks so much better IMO